public/app/plugins/datasource/tempo/QueryEditor/QueryField.tsx

```diff
@@ -192,6 +192,7 @@ function SearchSection({ logsDatasourceUid, onChange, onRunQuery, query }: Searc
           onRunQuery={onRunQuery}
           query={query.linkedQuery ?? ({ refId: 'linked' } as LokiQuery)}
           history={[]}
+          setQueryStats={() => {}}
```

Comment:
Again, I would remove all these noop function changes, it's far from an ideal component API.